slack/tasks.py

```diff
@@ -29,8 +29,9 @@ def process_event(data):
     # noinspection PyBroadException
     try:
         if data['event'].get('subtype') in SUBTYPE_BLACKLIST:
-            logger.debug(f'Ignoring {data["event"]["event_id"]}, subtype was in blacklist')
-            return
+            return logger.debug(f'Ignoring {data["event"]["event_id"]}, subtype was in blacklist')
```

Comment:
I've gotten KeyErrors on event_id before, just fyi

Reply:
Ill just call get on the dict, its a log anyway.